command_line_assistant/daemon/http/session.py

```diff
@@ -32,8 +33,16 @@ def get_session(config: Config) -> Session:
     """
     session = Session()
 
-    # Include the proxies defined by the user. By default, nothing is loaded.
+    # Include the proxies defined by the user. By default, environment variables is loaded.
     session.proxies.update(config.backend.proxies)
+    if not (config.backend.proxies):
+        http_proxy = os.environ.get("http_proxy")
+        if http_proxy:
+            session.proxies["http"] = http_proxy
+
+        https_proxy = os.environ.get("https_proxy")
+        if https_proxy:
+            session.proxies["https"] = https_proxy
```

Comment:
Would you mind moving this logic to https://github.com/rhel-lightspeed/command-line-assistant/blob/main/command_line_assistant/config/schemas/backend.py#L42 and overriding the `proxies` accordingly?